integration/systemd/load-and-verify-agent.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+# 
+#     http://aws.amazon.com/apache2.0/
+# 
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This wrapper script ensures that we've loaded the desired ECS agent
+# and have attached the "latest" tag to it.
+# USAGE: load-and-verify-agent.sh DEFAULT_TAG
```

Comment:
Just for my own understanding, when/where is this file supposed to be used?

Reply:
The intent was to make this part of the agent in-place upgrade workflow. It's not used in the unit file currently because I wasn't able to come up with the right way to represent the existing workflow in systemd.